src/commands/emulators-start.ts

```diff
@@ -18,6 +22,46 @@ module.exports = new Command("emulators:start")
     }
 
     utils.logLabeledSuccess("emulators", "All emulators started, it is now safe to connect.");
+    const table = new Table({
+      head: ["Emulator", "Host:Port", "View in Browser"],
```

Comment:
One more thing... We should drop the "View in Browser" heading altogether GUIs is not started.